lib/bsdlib/nrf91_sockets.c

```diff
@@ -147,6 +150,38 @@ static int z_to_nrf_optname(int z_in_level, int z_in_optname,
 		}
 		break;
 
+	case SOL_DFU:
+		switch (z_in_optname) {
+		case SO_DFU_FW_VERSION:
+			*nrf_out_optname = NRF_SO_DFU_FW_VERSION;
+			break;
+		case SO_DFU_RESOURCES:
```

Comment:
Good to have CI. It seems there is a typo in `fw-nrfconnect-zephyr`

Reply:
Updated Zephyr SHA to include fix.